src/Components/TableCharacters/TableCharacters.js

```diff
@@ -5,18 +5,32 @@ export default function TableCharacters(props) {
     return (
         <table className="table-caracter">
             <thead>
-                <th className="th-caracter" scope='col'>Pseudo</th>
-                <th className="th-caracter" scope='col'>Classe</th>
-                <th className="th-caracter" scope='col'>Age</th>
+                <tr>
+                    <th className="th-caracter" scope='col'>Date</th>
+                    <th className="th-caracter" scope='col'>Nom</th>
+                    <th className="th-caracter" scope='col'>Race</th>
+                    <th className="th-caracter" scope='col'>Classe</th>
+                    <th className="th-caracter" scope='col'>Carriere</th>
+                    <th className="th-caracter" scope='col'>Age</th>
+                    <th className="th-caracter" scope='col'>Taille</th>
+                    <th className="th-caracter" scope='col'>Cheveux</th>
+                    <th className="th-caracter" scope='col'>Yeux</th>
+                </tr>
             </thead>
             <tbody>
-                {props.charactersList.map((caracter) => {
+                {props.charactersList.map((character) => {
                     return (
-                        <tr>
-                            <td>{caracter.pseudo}</td>
-                            <td>{caracter.classe}</td>
-                            <td>{caracter.age}</td>
-                            <td><button className='btn-delete-table' onClick={() => (props.deleteCaracter(caracter.id))}> Del </button></td>
+                        <tr key={character.id}>
+                            <td>{character.date}</td>
+                            <td>{character.nom}</td>
+                            <td>{character.race}</td>
+                            <td>{character.classe}</td>
+                            <td>{character.carriere}</td>
+                            <td>{character.age} ans</td>
+                            <td>{character.taille} cm</td>
+                            <td>{character.cheveux}</td>
+                            <td>{character.yeux}</td>
+                            <td><button className='btn-delete-table' onClick={() => (props.deleteCaracter(character.id))}> Del </button></td>
```

Comment:
Essaye de préfixer tes actions reçues en props avec "on".

```suggestion
                            <td><button className='btn-delete-table' onClick={() => (props.onDelete(character.id))}> Del </button></td>
```